bcryptjs/index.d.ts

```diff
@@ -16,11 +16,19 @@ export declare function setRandomFallback(random: (random: number) => number[]):
  * Synchronously generates a salt.
  * @param  rounds Number of rounds to use, defaults to 10 if omitted
  * @return Resulting salt
+ * @throws If a random fallback is required but not set
```

Comment:
@andy-ms What about this throw-decorator? I'm asking, because this one is not related to the type annotations.
https://github.com/dcodeIO/bcrypt.js#gensaltsyncrounds-seed_length